tests/features/contact_form/submit_contact_form.feature

```diff
@@ -28,8 +28,23 @@ Feature: Submit the contact form
     # The honeypot field that needs to be empty on submission.
     Then the following fields should be present "user_homepage"
     And I should see the text "Submissions of this form are processed by a contractor of the European Commission."
+    And I fill in the following:
+      | First name     | Wolverine                   |
+      | Last name      | Wolves                      |
+      | Organisation   | The Deaf-Mute Wolves        |
+      | E-mail address | oswine@example.pt           |
+      | Category       | other                       |
+      | Subject        | Screen reader accessibility |
+      | Message        | Dear sir, madam, ...        |
+    And I attach the file "logo.png" to "Attachment"
+    # We need to wait 5 seconds for the spam protection time limit to pass.
+    And I wait for the spam protection time limit to pass
+    And I press "Submit"
+    Then I am on "collections"
```

Comment:
I've already explained that: This is wrong. We should **assert** that we're on `collections` and **not navigate** to `collections`. If you're unsure about what a step is doing, just hold ⌘ (macOs) and click on the step. It will drive you to the step definition.